dev/cloud/naming_conventions.md

```
@@ -0,0 +1,259 @@
+|                                            |     |     
+| ------------------------------------------ | --- | 
+| Request has body                           | No  |     
+| Successful response has body               | Yes |     
+| Safe (Does not change state of the Server) | Yes |     
+| Idempontent                                | Yes |     
+| Cacheable                                  | Yes |     
+A __Get__ Request should always be used when a single resource or an query without a body is used.
+Queries with a body should be avoided if possible (with the exception of GQL).
+
+##### POST
+|                                            |     |     
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)